### DISCUSSION TITLE
DSO catalog, series 4.x

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#2317] [Ideas]
I want to start the discussion for preparation fourth series of our DSO catalog. The new major version of catalog is reflected to changes the format of data and catalog will contain 17 columns:
1. (int)	- deep-sky object identificator
2. (float)	- RA (decimal degrees; J2000.0)
3. (float)	- Dec (decimal degrees; J2000.0)
4. (float)	- B magnitude
5. (float)	- V magnitude
6. (string)	- Object type (G, GX, GC, OC, NB, PN, DN, RN, C+N, HA, HII, SNR, BN, EN, SA, SC, RG, CL, IG, QSO, etc. or empty)
7. (string)	- Morphological type of object
8. (float)	- Major semi-axis size or radius (arcmin)
9. (float)	- Minor semi-axis size (arcmin)
10. (int)	- Orientation angle (degrees)
11. (float)	- Redshift
12. (float)	- Error of redshift
13. (float)	- Parallax (mas)
14. (float)	- Error of parallax (mas)
15. (float)	- Non-redshift distance (kpc)
16. (float)	- Error of non-redsift distance (kpc)
17. (string) - Array of designations

Of course, changes in the format will caused changes in Nebula subsystem in the Stellarium to allow support the synonyms and doubles and reduce the LOC for manage different catalogs.

Well, first 2 question to discussion:
1. Which catalogs should be included and which source of them?
2.  Should we support multiple types of objects or not (columns 6 and 7)?

**Comment by silas1037** [comment]
> Which catalogs should be included and which source of them?

Now with array of designations, we can enable extension and components of NGC/IC objs, like NGCNNNNA, ICNNNN-2;
Add important duplicate ID for the same catalog, such as NGC1269=NGC1291.
Many open cluster from other catalog (Basel, Berk, Biur, Blanco, Bochum...), I always find them in SAC database and *Catalogue of Star Clusters and Associations*. The same for some special PN names.

[Refer](http://www.danilopivato.com/catalogue/catalogues_list/catalogues_list.html).

**Reply by silas1037** [reply]
And, it's a pity to abandon Abell PN catalog, we can rename Abell(Galaxy Clusters) to ACO.

**Comment by gzotti** [comment]
Please make sure to specify 2&3 are J2000 coordinates,
And 8&9 must be radii or semi-axes consistently, not diameters. 
My feeling about double objects is that two catalogs may have two distinct descriptions for objects, so we should not throw them together. Of course, this may make click-identification difficult, but searching should always retrieve the object and its description from the target catalog if they differ in (6)type. One thing to specify and document as well is the actual data source we use for multiple identifications. I.e., what is the actual source of the sizes/magnitudes/types.

**Reply by alex-w** [reply]
> Please make sure to specify 2&3 are J2000 coordinates,
> And 8&9 must be radii or semi-axes consistently, not diameters.

Yes, of course! Thanks for suggestion.

> My feeling about double objects is that two catalogs may have two distinct descriptions for objects, so we should not throw them together. Of course, this may make click-identification difficult, but searching should always retrieve the object and its description from the target catalog if they differ in (6)type. One thing to specify and document as well is the actual data source we use for multiple identifications. I.e., what is the actual source of the sizes/magnitudes/types.

I disagree - UGC, PGC and NGC may contains little different data (e.g. morphological type) for same object and we have here 3 aliases for one object - not a 3 different objects in same place.

**Comment by ultrapre** [comment]
About 'Morphological type of object' of planetary nebula, it is a good choise to add Vorontsov-Velyaminov type for them. It describes forms of PNs well. I always find it at SAC deep-sky database.

**Reply by alex-w** [reply]
Please share the link or catalog name

**Reply by ultrapre** [reply]
> Please share the link or catalog name

Saguaro Astronomy Club Database version 8.1
https://www.saguaroastro.org/sac-downloads/

**Comment by ultrapre** [comment]
About odd Named Open Clusters, https://cdsarc.cds.unistra.fr/viz-bin/cat/VII/92A may be a good source for including many infomations.

**Comment by MOJNICK** [comment]
I'm missing M31 globulars.

**Comment by alex-w** [comment]
2 missing columns:
(string) - year of discovery
(string) - discoverer

**Reply by ultrapre** [reply]
Some data of them could be easily  found in Revised NGC project by Dr. Wolfgang Steinicke. But be careful, many objects between it and here are not the same one, so you could not use it absolutely correctly, unless someone crossing-identifies them like http://haroldcorwin.net/.

**Comment by Xio1996** [comment]
Hi,
Would a Surface Brightness column be needed or does Stellarium calculate that based on magnitude and major/minor axis?

Have you seen the [GLADE+](http://glade.elte.hu) catalogue. It brings together data from 6 other galaxy/quasar catalogues and provides some interesting information. Also available at [VizieR](https://cdsarc.cds.unistra.fr/viz-bin/cat/VII/291).

**Reply by alex-w** [reply]
> Would a Surface Brightness column be needed or does Stellarium calculate that based on magnitude and major/minor axis?

we calculated a surface brightness
